star-rx72n-firmware/lib/rx_hcsr04/src/rx_hcsr04_hal_mock.c

```
@@ -12,51 +12,255 @@
  */
 
+/**
+ * @brief Delay execution by the specified number of microseconds
+ *
+ * @param[in] us Delay in microseconds. k_hcsr04_delay_none performs a no-op.
+ *               Values above k_hcsr04_delay_max_us are clamped with a warning.
+ *
+ * @details Uses snprintf for warning formatting when clamping occurs.
+ *          This mock provides no real-time guarantees beyond invoking mock_delay_us().
+ */
 void hcsr04_hal_delay_us(uint32_t us)
```

Clang (14.0.6)

[warning] parameter name 'us' is too short, expected at least 3 characters

(readability-identifier-length)